contracts/Frencapital.sol

```diff
@@ -32,4 +32,8 @@ contract Frencapital is Ownable, ERC1155 {
         _mint(a, FREN, amount, "");
         _mint(this.owner(), FREN, (amount * (daoShare * 1000)) / 100000, "");
     }
+
+    function burn(address a, uint amount) public onlyOwner {
+        _burn(a, FREN, amount);
```

Comment:
Misschien nog zoiets toevoegen?

```suggestion
        _burn(a, FREN, amount);
        if (this.balanceOf(a, FREN) == 0) {
            participants--;
        }
```

Reply:
Wel overwogen, maar ik wil niet opnieuw die NFT uitdelen. Dus als we van 11 naar 10, naar 11 participanten gaan kom je in de knel. Ook denk ik dat we de hike in percentage niet terug willen draaien als iemand burned.

Reply:
In mijn nieuwe mint functie krijg je de NFT niet opnieuw uitgedeeld door check op `balance == 0` voor uitdelen, ik zal de PR pushen (zodra ik rechten heb). Bovendien is daar ook de hike in percentage uit.

Reply:
Hah, mooi, dan mergen we deze en dan voeg jij dit toe? Of jij zet t in de nieuwe code?

Reply:
Ik kan mijn PR in de jouwe mergen wellicht, moet nog even tests fixen.